lib/firefox_profile.js

```diff
@@ -603,7 +602,7 @@ FirefoxProfile.prototype._installExtension = function (addon, cb) {
     self = this;
 
   if (addon.slice(-4) === '.xpi') {
-    tmpDir = this._createTempFolder(addon.split(path.sep).slice(-1));
+    tmpDir = this._createTempFolder(addon.split(path.sep).pop());
```

Comment:
.slice() returned an array. It still worked because `'string' + array` would trigger `array.toString()`, but now I'm only using `path.join` to create the path